src/qgis_plugin_dev_tools/config/__init__.py

```diff
@@ -47,6 +51,19 @@ def __init__(
             distribution(Requirement(spec).name) for spec in runtime_requires
         ]
         self.changelog_file_path = changelog_file_path
+        self.append_distributions_to_path = append_distributions_to_path
+
+        if append_distributions_to_path:
```

Comment:
@komima what do you think, should this be included in all cases?

Reply:
this is now obsolete since the new `if auto_add_recursive_runtime_dependencies:` will control that block, not the fact if danger-style vendoring is used?